src/helpers/hotkeysManager.js

```diff
@@ -246,6 +250,19 @@ WebViewer(...)
           core.setCurrentPage(currPageNumber + 1);
         }
       },
+      /**
+       * Hold to switch between AnnotationEdit mode and Pan Mode
+       * @name WebViewer.Hotkeys#Space
+       */
+      space: e => {
+        e.preventDefault();
+        if (e.type === 'keyup') {
+          setToolModeAndGroup(store, 'AnnotationEdit', '');
```

Comment:
Hmm I think what we want is to switch to Pan temporarily. For example, if you are using a freehand tool, you can switch to the Pan tool by holding the space, but after you release the space key the tool should be switched back to the freehand tool. 

You can check the behaviour in the legacy UI here https://www.pdftron.com/samples/web/samples/viewing/viewing-with-legacy-ui/